ForumProject/subscriptions/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from startups.models import Startup
+from investors.models import Investor
+
+
+class SubscribeInvestorStartup(models.Model):
+    """
+    Represents the relationship between an investor and a startup they follow.
+
+    Attributes:
+        investor (Investor): The investor following the startup.
+        startup (Startup): The startup being followed by the investor.
+        saved_at (DateTimeField): The datetime when the investor saved the startup.
+    """
+
+    investor = models.ForeignKey(Investor, on_delete=models.CASCADE)
+    startup = models.ForeignKey(Startup, on_delete=models.CASCADE)
```

Comment:
The use of `models.ForeignKey` with `on_delete=models.CASCADE` is appropriate here, as it helps maintain data integrity by preventing orphan records. However, consider if this is the desired behavior, as sometimes you might want to keep historical data for reporting or analytics. In such cases, you might opt for `models.SET_NULL` with the fields set to allow nulls, or another strategy based on business needs of the project

Reply:
Done, thanks!